deploy/helm/sumologic/templates/logs/collector/otelcol/daemonset.yaml

```diff
@@ -67,6 +67,10 @@ spec:
       {{- else }}
       priorityClassName: {{ include "sumologic.metadata.name.priorityclass" $ctx | quote }}
       {{- end }}
+{{- if $.Values.sumologic.pullSecrets }}
```

Comment:
does the service account for the otelcol logs collector also need this option? Like here: https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/deploy/helm/sumologic/templates/serviceaccount.yaml

Reply:
we use different `serviceaccount` for `collection-sumologic-otelcol-logs-collector` https://github.com/SumoLogic/sumologic-kubernetes-collection/blob/main/deploy/helm/sumologic/templates/logs/collector/otelcol/serviceaccount.yaml
I can move adding  `imagePullSecrets` for to the `serviceaccount` to have consistency